browser/src/map/Clipboard.js

```diff
@@ -1268,7 +1306,31 @@ L.Clipboard = L.Class.extend({
 		var box = document.getElementById(id + '-box');
 		var innerDiv = L.DomUtil.create('div', '', null);
 		box.insertBefore(innerDiv, box.firstChild);
-		innerDiv.innerHTML = msg;
+
```

Comment:
I'm sorry that I joined late to the party. This chunk of the code is sub-optimal from the localization perspective. It is not possible to localize sentences broken in two pieces. 

Part 1: Please press
Part 2: to see more options

E.g. how to translate "Please press", if in some language the sentence translates to "To see more options please press <something>". Impossible.

Reply:
@gokaysatir would it be reasonable to revert just this part *and* add a comment that content is built this way for l10n reasons, so the next person won't do the same mistake? Thanks.

Reply:
Well, this makes the issue complicated :) I will find another solution that satisfies both l10 and CSP sides.